src/Components/Artwork/FillwidthItem.tsx

```diff
@@ -141,7 +141,7 @@ export class FillwidthItemContainer extends React.Component<
             />
           </ImageLink>
 
-          <Badge artwork={artwork} width={this.imageWidth} />
+          {showExtended && <Badge artwork={artwork} width={this.imageWidth} />}
```

Comment:
Don't show the weird buy now badges on artwork bricks:

<img width="193" alt="Screen Shot 2020-03-26 at 3 59 02 PM" src="https://user-images.githubusercontent.com/236943/77704630-bdd88b80-6f7a-11ea-8de4-092d091f2c3d.png">